src/lib/linux-api/src/signal.rs

```diff
@@ -1393,20 +1394,23 @@ pub type stack_t = linux_stack_t;
 
 // bindgen fails to bind this one.
 // Copied from linux's include/uapi/linux/signal.h.
-pub const LINUX_SS_AUTODISARM: i32 = 1 << 31;
+pub const LINUX_SS_AUTODISARM: u32 = 1 << 31;
 bitflags::bitflags! {
+    // While the Linux header defines the only current value of this flag as
+    // u32, the field in `stack_t::ss_flags` where this type is used is i32.
     #[repr(transparent)]
     #[derive(Copy, Clone, Debug, Default, Eq, PartialEq)]
     pub struct SigAltStackFlags: i32 {
-        const SS_AUTODISARM = LINUX_SS_AUTODISARM;
+        // The raw u32 value wraps around to a negative i32.
+        const SS_AUTODISARM = i32_from_u32_allowing_wraparound(LINUX_SS_AUTODISARM);
```

Comment:
Should `LINUX_SS_AUTODISARM` be non-pub if we're exposing it through `SigAltStackFlags`?

Reply:
It's currently exposed for our re-export to C